source/css/sass/patterns/organisms/content-header.scss

```diff
@@ -13,12 +14,11 @@
   @include blg-spacing("block-end", "small");
 
   &:after {
-    border-bottom: 1px solid $color-text-dividers;
-    border-block-end: 1px solid $color-text-dividers;
+    @include logical-property(border, block-end, 1px solid $color-text-dividers);
```

Comment:
This now gets `rem`-ed... maybe should be optional?

Reply:
Agree `rem`ming should be optional as this could be used with something where pixel values are desired (e.g. images).